docs/htdocs/info/docs/tools/vep/vep_formats.html

```diff
@@ -377,7 +377,7 @@ <h2 id="defaultout"> Default VEP output</h2>
         <li><strong>Position in protein</strong> - relative position of amino acid in protein</li>
         <li><strong>Amino acid change</strong> - only given if the variant affects the protein-coding sequence</li>
         <li><strong>Codon change</strong> - the alternative codons with the variant base in upper case</li>
-        <li><strong>Co-located variation</strong> - known identifier of existing variant</li>
+        <li><strong>Co-located variation</strong> - known identifier of existing variant. See <a href="/info/docs/tools/vep/script/vep_other.html#colocated">this page</a> for more details.</li>
```

Comment:
The original language is a little strange here. We can't report an unknown identifier!
How about-
- identifier of any <a href="/info/docs/tools/vep/script/vep_other.html#colocated">existing variants</a>. Switch on with <a href="/info/docs/tools/vep/script/vep_options.html#opt_check_existing"> --check_existing </a>

Reply:
It's changed.